src/configurations.jl

```diff
@@ -871,6 +873,38 @@ for O in [Orbital, RelativisticOrbital]
     @eval get_noble_gas(::Type{<:$O}, k) = noble_gases_configurations[$O][k]
 end
 
+"""
+    nrconfiguration(c::Configuration{<:RelativisticOrbital}) -> Configuration{<:Orbital}
```

Comment:
Not sure if it's the best name for this, but it's the one that popped into my head.

Reply:
`nonrelconfiguration` FTW